fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/client/ClientPersonImportHandler.java

```diff
@@ -95,7 +97,7 @@ private ClientData readClient(final Workbook workbook, final Row row, final Stri
         if (staffId == 0L) {
             staffId = null;
         }
-        String externalId = ImportHandlerUtils.readAsString(ClientPersonConstants.EXTERNAL_ID_COL, row);
+        ExternalId externalId = ExternalIdFactory.produce(ImportHandlerUtils.readAsString(ClientPersonConstants.EXTERNAL_ID_COL, row));
```

Comment:
Would it make more sense to use ExternalIdFactory.create()? it would auto generate one if the configuration is enabled and value was not provided

Reply:
Done